simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/network/CardCreatorService.kt

```diff
@@ -57,7 +57,7 @@ internal interface CardCreatorService {
       val logging = run {
         val httpLoggingInterceptor = HttpLoggingInterceptor()
         httpLoggingInterceptor.apply {
```

Comment:
This call to apply appears to be extraneous as currently written. I'd recommend changing to the following:

```
val httpLoggingInterceptor = HttpLoggingInterceptor().apply {
    level = HttpLoggingInterceptor.Level.BASIC
}
```